sdk/authorization/Azure.ResourceManager.Authorization/tests/Scenario/DenyAssignmentCollectionTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Resources;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Authorization.Tests.Scenario
+{
+    public class DenyAssignmentCollectionTests : AuthorizationManagementTestBase
+    {
+        public DenyAssignmentCollectionTests(bool isAsync)
+                    : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        public ResourceGroupResource ResourceGroup { get; set; }
+
+        [Test]
+        public async Task Get()
+        {
+            var randomGuid = Recording.Random.NewGuid().ToString();
+            var scope = DefaultSubscription.Id;
+            var denyAssignmentCollection = Client.GetDenyAssignments(scope);
+            try
+            {
+                var denyAssignment = (await denyAssignmentCollection.GetAsync(randomGuid)).Value;
+                Assert.AreEqual(1, 0);
```

Comment:
what is this always fail assertion for?

Reply:
If I want to test the get method, there must be an existing `denyAssignmentId`, but the list of DenyAssignments in the Azure portal is `empty`, so I cannot obtain the `denyAssignmentId`. Therefore, I can only test whether the get method will pass a non-existent Guid.

`Assert AreEqual (1, 0); `The function of is to terminate the program.
`Assert AreEqual (1, 1); `It is used to ensure that the program continues to run after obtaining the 404 error message.

Now I have deleted these two pieces of code and changed to a new format.